etc/impactx/perlmutter-nersc/batch_perlmutter.sh

```diff
@@ -9,39 +9,24 @@
 #SBATCH -t 01:00:00
 #SBATCH -N 4
 #SBATCH -J WarpX
+#    note: <proj> must end on _g
 #SBATCH -A <proj>
+#SBATCH -q regular
 #SBATCH -C gpu
 #SBATCH -c 32
 #SBATCH --ntasks-per-node=4
```

Comment:
Update: https://github.com/ECP-WarpX/WarpX/pull/3375

@n01r can you please update this after your test?